packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -93,3 +93,9 @@
 	}
 }
 
+.edit-site-global-styles__color-indicator-with-margin {
```

Comment:
Should this in itself be a component so we don't have bespoke CSS? Not necessarily a blocker, just curious.

Reply:
If the component you're referring to is a higher-level `ItemGroup` displaying a `ColorIndicator` + a text label, my suggestion would be the same as in https://github.com/WordPress/gutenberg/pull/39982#issuecomment-1087396110 .

If, otherwise, you're referring to creating a "color indicator with vertical margin" component, I'd say that we shouldn't do that — I think that margin rules should never be included in components.

_Side note: at some point I'd like to work on a `ColorSwatch` component (capacity permitting) that can replace/merge the use cases that today are covered by [`ColorIndicator`](https://wordpress.github.io/gutenberg/?path=/story/components-colorindicator--default) and `CircularOptionPicker` (used in [`ColorPalette`](https://wordpress.github.io/gutenberg/?path=/story/components-colorpalette--default)). Some high-level details were outlined in #35093._

Reply:
Going to mark this conversation as resolved since we decided to adopt the approach suggested in https://github.com/WordPress/gutenberg/pull/39982#discussion_r841929218